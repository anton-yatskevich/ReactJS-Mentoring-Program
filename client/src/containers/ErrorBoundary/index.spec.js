import React from 'react';
import { shallow } from 'enzyme';
import ErrorHandlerComponent from './index';

describe('ErrorHandler component', () => {
    it('should be rendered with default props', () => {
        const ErrorHandler = shallow(<ErrorHandlerComponent />);
        expect(ErrorHandler).toMatchSnapshot();
    });
});
