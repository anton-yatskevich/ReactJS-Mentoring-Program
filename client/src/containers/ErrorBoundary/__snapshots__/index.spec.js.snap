// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorHandler component should be rendered with default props 1`] = `<Component />`;
